tools/buildtools/xbuild/xbuild.go

```diff
@@ -95,14 +96,17 @@ func (xbuild Model) buildCommands() []string {
 		cmdSlice = append(cmdSlice, fmt.Sprintf("/target:%s", xbuild.target))
 	}
 
-	cmdSlice = append(cmdSlice, fmt.Sprintf("/p:SolutionDir=%s", filepath.Dir(xbuild.SolutionPth)))
+	// According to official docs this value should include the trailing backslash:
+	// https://docs.microsoft.com/en-us/cpp/build/reference/common-macros-for-build-commands-and-properties?view=vs-2019
+	solutionDirPth := ensureTrailingPathSeparator(filepath.Dir(xbuild.SolutionPth))
```

Comment:
NOTE: what if you merge `filepath.Dir(xbuild.SolutionPth)` in the extracted function and you call the functione something like `ensureSolutionDirPath`? The reason I am asking is because you can move the comments also to the given extracted function, it maybe more nice